lib/v2/fx-markets/channel.js

```diff
@@ -0,0 +1,59 @@
+const cheerio = require('cheerio');
+const date = require('@/utils/date');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const channel = ctx.params.channel;
+    const link = `https://www.fx-markets.com/${channel}`;
+    const html = (await got.get(link)).data;
+    const $ = cheerio.load(html);
+    const pageTitle = $('header.select-header > h1').text();
+    const title = `FX-Markets ${pageTitle}`;
+
+    const items = $('div#listings').children();
+    const articles = items
+        .map((i, el) => {
+            const $el = $(el);
+            const $titleEl = $el.find('h5 > a');
+            const articleURL = `https://www.fx-markets.com${$titleEl.attr('href')}`;
+            const articleTitle = $titleEl.attr('title');
+            return {
+                title: articleTitle,
+                link: articleURL,
+                pubDate: date($el.find('time').text()),
```

Comment:
Please use `parseDate` instead
```suggestion
                pubDate: parseDate($el.find('time').text()),
```